doc/man/vaulted.1

```diff
@@ -71,3 +71,41 @@ Exit code	Meaning
 65	There was an unrecoverable problem with the vault file.
 79	Invalid password supplied.
 .TE
+.SH GUI Password Prompts
+.PP
+Although Vaulted tries to make sure you can redirect stdin and friends,
+sometimes it is still preferable to use a GUI\-based password prompt. Vaulted
+uses an askpass mechanism to do this. When configured, askpass programs can use
+a GUI\-based password prompt that is specific to the system.
+.PP
+Vaulted uses its askpass mechanism when the \fB\fCVAULTED_ASKPASS\fR variable is set.
+When the executable file is invoked, the first parameter is prompt text to be
```

Comment:
Perhaps a bit more context in this area.  For starters, the VAULTED_ASKPASS variable is not clearly defined.  Additionally, the 'executable file' referenced here is not identified prior to stating it.  If this is a facet of the ASKPASS mechanism perhaps it should be confirmed as such?